File: iFlat/src/main/java/com/iflat/report/bean/wip/manhour/DetailOfMiscWo.java
package com.iflat.report.bean.wip.manhour;

import java.util.Date;

/**
 * Created by tyriv on 2015/11/12.
 */
public class DetailOfMiscWo {

    private String woId;
    private String type;
    private String reason;
    private String content;
    private double manhour;
    private String projectCode;
    private String projectName;
    private String orgName;
    private String teamName;
    private String groupName;
    private Date createTime;
    private String personInCharge;

    public double getManhour() {
        return manhour;
    }

    public void setManhour(double manhour) {
        this.manhour = manhour;
    }

    public String getWoId() {
        return woId;
    }

    public void setWoId(String woId) {
        this.woId = woId;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getProjectCode() {
        return projectCode;
    }

    public void setProjectCode(String projectCode) {
        this.projectCode = projectCode;
    }

    public String getProjectName() {
        return projectName;
    }

    public void setProjectName(String projectName) {
        this.projectName = projectName;
    }

    public String getReason() {
        return reason;
    }

    public void setReason(String reason) {
        this.reason = reason;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public String getOrgName() {
        return orgName;
    }

    public void setOrgName(String orgName) {
        this.orgName = orgName;
    }

    public String getTeamName() {
        return teamName;
    }

    public void setTeamName(String teamName) {
        this.teamName = teamName;
    }

    public String getGroupName() {
        return groupName;
    }

    public void setGroupName(String groupName) {
        this.groupName = groupName;
    }

    public Date getCreateTime() {
        return createTime;
    }

    public void setCreateTime(Date createTime) {
        this.createTime = createTime;
    }

    public String getPersonInCharge() {
        return personInCharge;
    }

    public void setPersonInCharge(String personInCharge) {
        this.personInCharge = personInCharge;
    }
}

File: web/app/model/report/wip/manhour/DetailOfMiscWo.js
Ext.define('iFlat.model.report.wip.manhour.DetailOfMiscWo', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'woId', type: 'string'},
        {name: 'type', type: 'string'},
        {name: 'reason', type: 'string'},
        {name: 'content', type: 'string'},
        {name: 'manhour', type: 'integer'},
        {name: 'projectCode', type: 'string'},
        {name: 'projectName', type: 'string'},
        {name: 'orgName', type: 'string'},
        {name: 'teamName', type: 'string'},
        {name: 'groupName', type: 'string'},
        {name: 'createTime', type: 'date'},
        {name: 'personInCharge', type: 'string'},
    ]
})
